Hypertext Assassins/index.html

```diff
@@ -0,0 +1,514 @@
+                <a href="#" class="button">Explore more</a>
+            </div>
+            <article class="right_container">
+                <div class="col-9">
+                    <div class="p-2 flex-fill  bd-highlight margin-down">
+                        <i class="fas fa-bolt icon-size"></i>
+                        <h6 class="h">Fast & effective</h6>
+                        <p class="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                    <div class="p-2 flex-fill bd-highlight">
+                        <i class="far fa-user icon-size"></i>
+                        <h6 class="h">Consulting</h6>
+                        <p class="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+            </article>
+            <article class="d-flex right_container">
+                <div class="col-9">
+                    <div class="p-2 flex-fill bd-highlight margin-down">
+                        <i class="fas fa-tv icon-size"></i>
+                        <h6 class="h ">Marketing strategy</h6>
+                        <p class="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                    <div class="p-2 flex-fill bd-highlight">
+                        <i class="fas fa-headphones icon-size"></i>
+                        <h6 class="h">Startup support</h6>
+                        <p class="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+            </article>
+        </article>
+    </section>
+
+    <!--Counters part *Shadan Abdulkareem*-->
+
+    <section class="third-part">
+        <article class="card">
+            <img src="./images/daniel-korpai-8GDCzWrcE3M-unsplash.jpg" class="card-img img-responsive "
+                style="height:600px;">
+            <div class="card-img-overlay text-light d-flex flex-column align-content-center justify-content-center text-center">
+                <div class="row ">
+                    <div class="col-lg-3 col-sm-6 col-mid-6 col-xs-6">
+                        <h1 class="font-weight-bold animated flip delay-1s counter">32</h1>
+                        <hr class="border-warning justify-content-around mt-2 bg-warning border-dark"
+                            style="width:17%;height:2px;">
+                        <h4 class="font-weight-light animated bounceInLeft slower">Businesses built in<br>
+                            just 2 years</h4>
+                    </div>
+                    <div class="col-lg-3 col-sm-6 col-mid-6 col-xs-6">
+                        <h1 class=" font-weight-bold animated flip delay-3s"><span class="counter">100</span>+</h1>
+                        <hr class="border-warning justify-content-around mt-2 bg-warning border-dark"
+                            style="width:17%;height:2px;">
```

Comment:
I saw you used this style multiple times so it would much better if it was in on class